e2e-tests/specs/all.e2e.js

```diff
@@ -252,41 +252,80 @@ if (args.includes('env=LOCAL')) {
     config.collection.endpoint = '../../mock-json/smoke.json';
 }
 
-describe('Hide CTA(s):', async () => {
-    it('CTA should exist', async () => {
+describe('Carousel Behaviors', async () => {
+    const cloneConfig = structuredClone(config);
+    cloneConfig.collection.layout.container = 'carousel';
+
+    const state = btoa(JSON.stringify(cloneConfig));
+    const url = `${serverPath}/html/e2e/carousel.html?state=${state}`;
+
+    // example
+    it('Carousel Title should exist', async () => {
+        await browser.url(url);
+        await browser.setTimeout({ script: 50000 });
+        const title = await $('.consonant-CarouselInfo-collectionTitle').getText();
+        await expect(title).toEqual('Lorem Ipsum');
+    });
+
+    // example
+    it('All cards should appear', async () => {
+        await browser.url(url);
+        await browser.setTimeout({ script: 50000 });
+        const grid = await $('.consonant-CardsGrid');
+        await expect(grid).toHaveChildren(8);
+    });
+
+    it('You should be able to click to the next page', async () => {
+        await browser.url(url);
+        await browser.setTimeout({ script: 50000 });
+        const nextButton = await $('.consonant-Button--next');
+        await nextButton.click();
+        const prevButton = await $('.consonant-Button--previous');
+        await expect(prevButton).toBeDisplayed();
+    });
+});
+
+describe('Card Behaviors', async () => {
+    // checking if time has passed upcoming becomes live
+});
+
+// contains good examples of changing the config and seeing the results
+describe('CTA Behaviors', async () => {
+    /* it('CTA should exist', async () => {
         const cloneConfig = structuredClone(config);
         const state = btoa(JSON.stringify(cloneConfig));
-        const url = `${serverPath}/html/e2e/MWPW-126169.html?state=${state}`;
+        const url = `${serverPath}/html/e2e/e2e-grid.html?state=${state}`;
         await browser.url(url);
         await browser.setTimeout({ script: 50000 });
-        const buttonText = await $('#ac578dee-f01b-3ea0-a282-2116619e4251 .consonant-BtnInfobit--cta').getText();
-        expect(buttonText).toEqual('Read More');
```

Comment:
Looks like this is removed. Could you please keep it inside the comments? 
expect(buttonText).toEqual('Read More');